### ISSUE TITLE
Spot testing on iOS 9.3 preview?

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/tasks#472]
@phet-steele can you see if you can put the iOS 9.3 preview on one of the iPad 2's and do some spot testing on published sims?

Ask @oliver-phet if you need access to the developer account

**Comment by ariel-phet** [comment]
@BryceAG see if you can test this out

**Comment by ariel-phet** [comment]
Test a bunch a bunch of sims, try downloading and putting in the reading list.

**Comment by BryceAG** [comment]
I have updated Tycho to the 9.3 beta. So far I have downloaded Energy Skate Park Basics and Hooke's Law to the reading list and they work fine. I have also tested a few RC sims that were previously downloaded to the reading list (neuron, Faraday's Law) and they also seem to work, but Faraday's has a lot of lag, particularly with field lines on. Strangely enough, all string tests other than rtl improve the lag. There also are some Gravity Force Lab apps, an RC test and a dev test, that were already downloaded before the update. The RC test works, but the dev does not.

**Comment by ariel-phet** [comment]
@BryceAG - please try a few of these sims as published versions off the web (not downloading to the reading list) such as GFL and Faraday's Law.

**Comment by BryceAG** [comment]
will do

**Comment by BryceAG** [comment]
Molecular Shapes, Faraday's Law, and Gravity Force Lab have all been tested off the website and appear to be fully functional.

**Comment by ariel-phet** [comment]
@BryceAG one other request...can you check a couple sims with games (these hit a few different code libraries). Balancing Act, Graphing Lines, and Balancing Chemical Equations should cover it. Please test these directly off the website, thanks!

**Comment by BryceAG** [comment]
I have tested Balancing Act, Graphing Lines, and Balancing Chemical Equations off the website, and they function well. There was one issue I found with Balancing act, but it was consistent across multiple operating systems, so I did not refer to this task. It is phetsims/balancing-act#63

**Comment by ariel-phet** [comment]
Excellent, thanks @BryceAG, closing issue

**Comment by phet-steele** [comment]
@ariel-phet @oliver-phet I just got a pop up asking if I want to update to iOS 9.3 Beta 2, can I get a go ahead?

**Comment by ariel-phet** [comment]
@phet-steele absolutely. Feel free to also do some spot testing on Beta 2

**Comment by BryceAG** [comment]
I have tested energy skate park basics and hookes law from the reading list, and downloaded faradays law to the reading list. I have also tested molecular shapes and gravity force lab off the Web. And tested the games in balancing act, balancing chemical equations, and graphing  lines. They all appear to be functioning well with beta 2

**Comment by phet-steele** [comment]
Thanks @BryceAG

**Comment by phet-steele** [comment]
@BryceAG guess what? It's Beta 3 now! A day where there isn't much for you to do, can you redo these tests again? This would be pretty super low priority.

**Comment by BryceAG** [comment]
I have tested faradays law and hookes law from the reading list, deleted and redownloaded energy skate park basics to the reading list. I have also tested molecular shapes and gravity force lab off the Web. I also tested the games in balancing act, balancing chemical equations, and graphing lines. They all appear to be functioning well with beta 3

**Comment by phet-steele** [comment]
@BryceAG Thanks yet again!

**Comment by BryceAG** [comment]
Beta 6 has been released yesterday, so I missed 4 and 5, but... I have tested energy skate park basics and hookes law from the reading list, deleted and redownloaded Faraday's law to the reading list. I have also tested molecular shapes and gravity force lab off the Web. I also tested the games in balancing act, balancing chemical equations, and graphing lines. They all appear to be functioning well with beta 6
